### PR TITLE
Add file response

### PR DESCRIPTION
**Pull request by Iamsamwhoami**
This makes it possible to send a file response to Hedvig.

This little thing is cherry-picked from: https://github.com/HedvigInsurance/giraffe/pull/108

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -295,6 +326,57 @@ const registerDirectDebit = async (
...
+  )
+
+  if (!responseMessage) {
+    throw new Error("Tried to respond to a message that doesn't exist")
+  }
+
+  const responseMessageWithFile = {
+    ...responseMessage,
+    body: {
+      ...responseMessage.body,
```
Comment (palmfjord) [inline src/api/index.ts]:
```ts
...responseMessage.body,
...responseInput.body,
```
?

**Comment by Iamsamwhoami** [comment]
@palmfjord fixed 👀

File: src/api/index.ts [hunk 2]
```diff
@@ -120,13 +145,19 @@ const callApi: CallApi = async (url, options = {}) => {
   }
 
   const res = await fetch(`${config.BASE_URL}${url}`, requestOptions)
-  validateStatus(res)
+  await validateStatus(res)
```
Comment (palmfjord) [inline src/api/index.ts]: now that we're `await`ing this, shouldn't the rest of the `validateStatus` functions also be async? like this guy for example:  https://github.com/HedvigInsurance/giraffe/pull/117/files#diff-55e1fcd9400087f22bb1f21cfaf39385R250

Comment (Iamsamwhoami) [reply src/api/index.ts]: 🙃 didn't realize, removed!

**Pull request merged**